### PR TITLE
Updated coach mark tokens for both desktop and mobile

### PR DESCRIPTION
**Pull request by KayiuCarlson**
Updated coach mark tokens by creating new and deprecated old Title, Body, Pagination body font size tokens.

Added the word "font" in the font size token naming.

n/a

n/a

<li> [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue) </li>
<li> [x] Minor (add a new token, changing a value; non-breaking change which adds functionality) </li>
<li> [ ] Major (deleting a token, changing token value type; fix or feature that would cause existing functionality to change) </li>

## Checklist:

<li> [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html). </li>
<li> [x] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.) </li>

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/8574665635)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-body-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-pagination-body-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-title-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-pagination-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-pagination-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-title-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-title-font-size
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (23) in-field-button-edge-to-fill-small/desktop, in-field-button-edge-to-fill-extra-large/desktop, in-field-progress-circle-edge-to-fill/desktop, in-field-progress-circle-size-75/desktop, in-field-progress-circle-size-100/desktop, in-field-progress-circle-size-200/desktop, in-field-progress-circle-size-300/desktop, in-field-button-edge-to-fill-small/mobile, in-field-button-edge-to-fill-extra-large/mobile, in-field-progress-circle-edge-to-fill/mobile, in-field-progress-circle-size-75/mobile, in-field-progress-circle-size-100/mobile, in-field-progress-circle-size-200/mobile, in-field-progress-circle-size-300/mobile, combo-box-visual-to-field-button, in-field-button-edge-to-fill-medium, in-field-button-edge-to-fill-large, coach-mark-body-font-size/desktop, coach-mark-body-font-size/mobile, coach-mark-pagination-body-font-size/desktop, coach-mark-pagination-body-font-size/mobile, coach-mark-title-font-size/desktop, coach-mark-title-font-size/mobile
MODIFIED TOKENS: (7) coach-mark-body-size, coach-mark-pagination-body-size, coach-mark-title-size, combo-box-visual-to-field-button-extra-large, combo-box-visual-to-field-button-large, combo-box-visual-to-field-button-medium, combo-box-visual-to-field-button-small
DEPRECATED TOKENS: (19) combo-box-quiet-minimum-width-multiplier, combo-box-visual-to-field-button-small/desktop, combo-box-visual-to-field-button-small/mobile, combo-box-visual-to-field-button-medium/desktop, combo-box-visual-to-field-button-medium/mobile, combo-box-visual-to-field-button-large/desktop, combo-box-visual-to-field-button-large/mobile, combo-box-visual-to-field-button-extra-large/desktop, combo-box-visual-to-field-button-extra-large/mobile, combo-box-visual-to-field-button-quiet, coach-mark-title-size/desktop, coach-mark-title-size/mobile, coach-mark-body-size/desktop, coach-mark-body-size/mobile, coach-mark-pagination-body-size/desktop, coach-mark-pagination-body-size/mobile, in-field-button-edge-to-fill, drop-shadow-y, drop-shadow-blur
CHANGED VALUES: (8) combo-box-visual-to-field-button-extra-large/desktop, combo-box-visual-to-field-button-large/desktop, combo-box-visual-to-field-button-medium/desktop, combo-box-visual-to-field-button-quiet, combo-box-visual-to-field-button-small/desktop, coach-mark-body-size/desktop, coach-mark-pagination-body-size/desktop, coach-mark-title-size/desktop
CHANGED SCHEMAS: (5) combo-box-visual-to-field-button-extra-large, combo-box-visual-to-field-button-large, combo-box-visual-to-field-button-medium, combo-box-visual-to-field-button-quiet, combo-box-visual-to-field-button-small
```
</p></details>

TEST COMPLETE

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json [hunk 1]
```diff
@@ -1079,6 +1099,16 @@
       }
     }
   },
+  "coach-mark-title-font-size": {
+    "value": "{heading-size-xs}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json:1103]: @KayiuCarlson Looking at the token spec, I'm seeing the value is marked as `title-size-m`, could you double-check this?

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json [hunk 2]
```diff
@@ -999,6 +999,16 @@
       }
     }
   },
+  "coach-mark-body-font-size": {
+    "value": "{body-size-xs}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json:1003]: @KayiuCarlson In the token spec, I'm seeing this as `body-size-s`

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json [hunk 3]
```diff
@@ -1079,6 +1099,16 @@
       }
     }
   },
+  "coach-mark-title-font-size": {
+    "value": "{heading-size-xxs}",
```
Comment (lynnhao) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json:1103]: @KayiuCarlson this one I'm seeing as `title-size-s`

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/8601445439)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-body-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-pagination-body-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-title-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-pagination-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-pagination-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-title-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-title-font-size
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (23) in-field-button-edge-to-fill-small/desktop, in-field-button-edge-to-fill-extra-large/desktop, in-field-progress-circle-edge-to-fill/desktop, in-field-progress-circle-size-75/desktop, in-field-progress-circle-size-100/desktop, in-field-progress-circle-size-200/desktop, in-field-progress-circle-size-300/desktop, in-field-button-edge-to-fill-small/mobile, in-field-button-edge-to-fill-extra-large/mobile, in-field-progress-circle-edge-to-fill/mobile, in-field-progress-circle-size-75/mobile, in-field-progress-circle-size-100/mobile, in-field-progress-circle-size-200/mobile, in-field-progress-circle-size-300/mobile, combo-box-visual-to-field-button, in-field-button-edge-to-fill-medium, in-field-button-edge-to-fill-large, coach-mark-body-font-size/desktop, coach-mark-body-font-size/mobile, coach-mark-pagination-body-font-size/desktop, coach-mark-pagination-body-font-size/mobile, coach-mark-title-font-size/desktop, coach-mark-title-font-size/mobile
MODIFIED TOKENS: (7) coach-mark-body-size, coach-mark-pagination-body-size, coach-mark-title-size, combo-box-visual-to-field-button-extra-large, combo-box-visual-to-field-button-large, combo-box-visual-to-field-button-medium, combo-box-visual-to-field-button-small
DEPRECATED TOKENS: (19) combo-box-quiet-minimum-width-multiplier, combo-box-visual-to-field-button-small/desktop, combo-box-visual-to-field-button-small/mobile, combo-box-visual-to-field-button-medium/desktop, combo-box-visual-to-field-button-medium/mobile, combo-box-visual-to-field-button-large/desktop, combo-box-visual-to-field-button-large/mobile, combo-box-visual-to-field-button-extra-large/desktop, combo-box-visual-to-field-button-extra-large/mobile, combo-box-visual-to-field-button-quiet, coach-mark-title-size/desktop, coach-mark-title-size/mobile, coach-mark-body-size/desktop, coach-mark-body-size/mobile, coach-mark-pagination-body-size/desktop, coach-mark-pagination-body-size/mobile, in-field-button-edge-to-fill, drop-shadow-y, drop-shadow-blur
CHANGED VALUES: (8) combo-box-visual-to-field-button-extra-large/desktop, combo-box-visual-to-field-button-large/desktop, combo-box-visual-to-field-button-medium/desktop, combo-box-visual-to-field-button-quiet, combo-box-visual-to-field-button-small/desktop, coach-mark-body-size/desktop, coach-mark-pagination-body-size/desktop, coach-mark-title-size/desktop
CHANGED SCHEMAS: (5) combo-box-visual-to-field-button-extra-large, combo-box-visual-to-field-button-large, combo-box-visual-to-field-button-medium, combo-box-visual-to-field-button-quiet, combo-box-visual-to-field-button-small
```
</p></details>

TEST COMPLETE

**Review by lynnhao — APPROVED** [review]
Thank you!

**Review by karstens — CHANGES_REQUESTED** [review]
please see my comment here: https://adobedesign.slack.com/archives/GTZ7NSNR0/p1712749311910149?thread_ts=1712329548.189499&cid=GTZ7NSNR0

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2-non-colors](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/8646114802)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`
TOKENS STUDIO DATA IS NON-COLOR ONLY

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-body-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-pagination-body-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 coach-mark-title-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-pagination-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-pagination-body-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.coach-mark-title-font-size
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.coach-mark-title-font-size
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (23) in-field-button-edge-to-fill-small/desktop, in-field-button-edge-to-fill-extra-large/desktop, in-field-progress-circle-edge-to-fill/desktop, in-field-progress-circle-size-75/desktop, in-field-progress-circle-size-100/desktop, in-field-progress-circle-size-200/desktop, in-field-progress-circle-size-300/desktop, in-field-button-edge-to-fill-small/mobile, in-field-button-edge-to-fill-extra-large/mobile, in-field-progress-circle-edge-to-fill/mobile, in-field-progress-circle-size-75/mobile, in-field-progress-circle-size-100/mobile, in-field-progress-circle-size-200/mobile, in-field-progress-circle-size-300/mobile, combo-box-visual-to-field-button, in-field-button-edge-to-fill-medium, in-field-button-edge-to-fill-large, coach-mark-body-font-size/desktop, coach-mark-body-font-size/mobile, coach-mark-pagination-body-font-size/desktop, coach-mark-pagination-body-font-size/mobile, coach-mark-title-font-size/desktop, coach-mark-title-font-size/mobile
MODIFIED TOKENS: (7) coach-mark-body-size, coach-mark-pagination-body-size, coach-mark-title-size, combo-box-visual-to-field-button-extra-large, combo-box-visual-to-field-button-large, combo-box-visual-to-field-button-medium, combo-box-visual-to-field-button-small
DEPRECATED TOKENS: (21) color-loupe-drop-shadow-y, color-loupe-drop-shadow-blur, combo-box-quiet-minimum-width-multiplier, combo-box-visual-to-field-button-small/desktop, combo-box-visual-to-field-button-small/mobile, combo-box-visual-to-field-button-medium/desktop, combo-box-visual-to-field-button-medium/mobile, combo-box-visual-to-field-button-large/desktop, combo-box-visual-to-field-button-large/mobile, combo-box-visual-to-field-button-extra-large/desktop, combo-box-visual-to-field-button-extra-large/mobile, combo-box-visual-to-field-button-quiet, coach-mark-title-size/desktop, coach-mark-title-size/mobile, coach-mark-body-size/desktop, coach-mark-body-size/mobile, coach-mark-pagination-body-size/desktop, coach-mark-pagination-body-size/mobile, in-field-button-edge-to-fill, drop-shadow-y, drop-shadow-blur
CHANGED VALUES: (10) color-loupe-drop-shadow-blur, color-loupe-drop-shadow-y, combo-box-visual-to-field-button-extra-large/desktop, combo-box-visual-to-field-button-large/desktop, combo-box-visual-to-field-button-medium/desktop, combo-box-visual-to-field-button-quiet, combo-box-visual-to-field-button-small/desktop, coach-mark-body-size/desktop, coach-mark-pagination-body-size/desktop, coach-mark-title-size/desktop
CHANGED SCHEMAS: (7) color-loupe-drop-shadow-blur, color-loupe-drop-shadow-y, combo-box-visual-to-field-button-extra-large, combo-box-visual-to-field-button-large, combo-box-visual-to-field-button-medium, combo-box-visual-to-field-button-quiet, combo-box-visual-to-field-button-small
```
</p></details>

TEST COMPLETE

**Review by karstens — APPROVED** [review]

**Comment by karstens** [comment]
@lynnhao @KayiuCarlson Just FYI: This is how the tokens appear with the type "fontSizes".

Screenshot 2024-04-11 at 13 46 22

**Comment by karstens** [comment]
@GarthDB This is ready to merge. 👍

**Pull request merged**